src/signTx.h

```diff
@@ -16,10 +17,11 @@ typedef enum {
 	SIGN_STAGE_FEE = 26,
 	SIGN_STAGE_TTL = 27,
 	SIGN_STAGE_CERTIFICATES = 28,
-	SIGN_STAGE_WITHDRAWALS = 29,
-	SIGN_STAGE_METADATA = 30,
-	SIGN_STAGE_CONFIRM = 31,
-	SIGN_STAGE_WITNESSES = 32,
+	SIGN_STAGE_CERTIFICATES_PRC = 29, // pool registration certificate sub-machine
```

Comment:
what this "PRC" suffix stands for?

Reply:
Pool Registration Certificate

Reply:
I would just rename it to SIGN_STAGE_CERTIFICATES_POOL